node/services/businessProfileService.ts

```diff
@@ -75,10 +75,40 @@ export const businessProfileService = (params: {
     }
   }
 
+  const clearBusinessProfileData = async (): Promise<void> => {
+    try {
+      const appSettings = ((await params.apps.getAppSettings(
+        process.env.VTEX_APP_ID as string
+      )) ?? {}) as AppSettings
+
+      appSettings.catalogId = ''
+      appSettings.advertiserId = ''
+      appSettings.bcId = ''
+      appSettings.tradePolicyId = ''
+      appSettings.pixelCode = ''
+      appSettings.accessToken = ''
+      appSettings.status = 'disconnected'
```

Comment:
```suggestion
      appSettings.accessToken = ''
      appSettings.passPhrase = ''
      appSettings.status = 'disconnected'
```

I think you forgot the passphrase. 😄